Source/DataSources/CzmlDataSource.js

```diff
@@ -600,7 +600,10 @@ define([
         var hasInterval = defined(combinedInterval) && !combinedInterval.equals(Iso8601.MAXIMUM_INTERVAL);
 
         if (!isReference) {
-            referenceFrame = defaultValue(ReferenceFrame[packetData.referenceFrame], undefined);
+            if (defined(packetData.referenceFrame)) {
+                referenceFrame = ReferenceFrame[packetData.referenceFrame];
+            }
+            referenceFrame = defaultValue(referenceFrame, ReferenceFrame.FIXED);
```

Comment:
> OK, so if you put in an invalid value, you quietly get FIXED without seeing any error.

Just to be clear, this is not a change in behavior, it's always worked this way.  @shunter just sets it explicitly to fixed here rather than `undefined` getting passed into the load code later on.  CZML specifies fixed as default so this is technically more correct (and doesn't rely on the fact that fixed in is the default in Cesium).